petab/visualize/plotting_config.py

```diff
@@ -72,8 +72,9 @@ def plot_lowlevel(vis_spec: pd.DataFrame,
         # add xOffset
         conditions = conditions + vis_spec.xOffset[i_visu_spec]
 
-        # TODO sort mean and sd/sem by x values (as for simulatedData below)
+        # TO DO sort mean and sd/sem by x values (as for simulatedData below)
```

Comment:
Please keep as "TODO", that is what most tools recognize. "TO DO" is just regular text without special meaning.

Reply:
+ for multiline TODOs, add 2 blanks in the beginning of the continuation lines, then it is recognized as one single issue